shoes-swt/spec/shoes/swt/app_spec.rb

```diff
@@ -122,4 +124,36 @@
       it_behaves_like 'reports client area dimensions'
     end
   end
+
+  describe 'App Background color' do
+    it 'has the given background when specified' do
+      not_resizable = Shoes::Swt::App.new app_unresizable
+      background = not_resizable.shell.background
+      expect(background).to eq swt_salmon
+    end
+
+
+    it 'has the default system background when unspecified' do
+      allow(Shoes::App)
+        .to receive_messages(app_title: 'double',
+                             width: 0,
+                             height: 0,
+                             opts: { background: Shoes::COLORS[:system_background] })
+
+      default_background = ::Swt.display.getSystemColor(::Swt::SWT::COLOR_WIDGET_BACKGROUND)
+      app = Shoes::Swt::App.new(Shoes::App)
+      background = app.shell.background
+      expect(background).to eq default_background
+    end
```

Comment:
Oh yeah, thanks! You're absolutely right.

I switched that our for this now:

```ruby
app = Shoes::Swt::App.new(Shoes::InternalApp.new(Shoes::App.new, {}))
```

unfortunately the test is now failing because the app is using the fallback color: `[237, 237, 237]` instead of the swt color, which for me is `[232, 232, 232]`.

I'll look around and see if I can understand why.